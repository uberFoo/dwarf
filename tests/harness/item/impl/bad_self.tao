fn no_self(self) {}

fn main() {}
