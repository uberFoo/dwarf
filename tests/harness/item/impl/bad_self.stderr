[31mError:[0m self may only be used inside of an implementation block
   [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0mbad_self:1:12[38;5;246m][0m
   [38;5;246mâ”‚[0m
 [38;5;246m1 â”‚[0m [38;5;249mf[0m[38;5;249mn[0m[38;5;249m [0m[38;5;249mn[0m[38;5;249mo[0m[38;5;249m_[0m[38;5;249ms[0m[38;5;249me[0m[38;5;249ml[0m[38;5;249mf[0m[38;5;249m([0m[31ms[0m[31me[0m[31ml[0m[31mf[0m[38;5;249m)[0m[38;5;249m [0m[38;5;249m{[0m[38;5;249m}[0m
 [38;5;240m  â”‚[0m            [31mâ”€[0m[31mâ”€[0m[31mâ”¬[0m[31mâ”€[0m  
 [38;5;240m  â”‚[0m              [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m used here
[38;5;246mâ”€â”€â”€â•¯[0m

